examples/models/llama2/model.py

```diff
@@ -150,16 +150,32 @@ def __init__(self, **kwargs):
             output_prune_map = {int(k): v for (k, v) in output_prune_map.items()}
         max_seq_len = self.max_seq_len
         max_batch_size = 1
+        print("params: ", params)
+        params.pop("rope_theta", None)
         model_args: ModelArgs = ModelArgs(
             max_seq_len=max_seq_len,
             max_batch_size=max_batch_size,
-            use_kv_cache=self.use_kv_cache,
-            use_sdpa_with_kv_cache_op=self.use_sdpa_with_kv_cache_op,
-            generate_full_logits=self.generate_full_logits,
-            output_prune_map=output_prune_map,
-            enable_dynamic_shape=self.enable_dynamic_shape,
+            # input_vocab_size=params["input_vocab_size"],
+            # use_kv_cache=self.use_kv_cache,
+            # use_sdpa_with_kv_cache_op=self.use_sdpa_with_kv_cache_op,
+            # generate_full_logits=self.generate_full_logits,
+            # output_prune_map=output_prune_map,
+            # enable_dynamic_shape=self.enable_dynamic_shape,
+            use_layer_norm_op=True,
```

Comment:
@chiwwang @shewu-quic here is the place to switch between layer norm and rms norm